vuepress/docs/v7.3/tutorials/solution/install-standard-demo.md

```diff
@@ -91,18 +86,16 @@ Install the Standard Banking Demo by integrating the Entando Cloud Hub into your
 4. **`Install` each bundle:**
 
    ::: warning
-   **Order of installation is important.** The `standard-demo-content-bundle` must be installed last, as it relies on MFEs from the other bundles to set up each page. To uninstall the Banking demo, the `standard-demo-content-bundle` must be uninstalled first to resolve dependencies before other bundles can be be removed.
+   **Order of installation is important.** The `sd-content-bundle` must be installed last, as it relies on MFEs from the other bundles to set up each page. To uninstall the Banking demo, the `sd-content-bundle` bundle must be uninstalled first to resolve dependencies before other bundles can be be removed.
```

Comment:
**The order of installation is important.** `sd-content-bundle` must be installed last, as it relies on MFEs from the other bundles to set up each page. To uninstall the Banking demo, `sd-content-bundle` must be uninstalled first to resolve dependencies before other bundles can be be removed.